docs/signer.md

```diff
@@ -0,0 +1,354 @@
+1. Build the tool.
+
+    ```shell
+    go build -o ./signer ./cmd/kritis/signer
+    ```
+
+2. Enable GCP services.
+
+    The signing tool needs to access a number of Google Cloud Platform (GCP) services.
+First we need to pick a GCP project and enable those services within the project.
+
+    1. Set the default GCP project used by `gcloud`.
+
+        ```shell
+        export PROJECT_ID=[PROJECT ID]
+        ```
+
+    2. Set the default GCP project used by `gcloud`.
+
+        ```shell
+        gcloud config set project ${PROJECT_ID}
+        ```
+
+    3. Run `gcloud` to enable services within a project.
+
+        ```shell
+        gcloud services enable \
+          cloudbuild.googleapis.com \
+          containerregistry.googleapis.com \
+          containeranalysis.googleapis.com \
+          containerscanning.googleapis.com \
+          cloudkms.googleapis.com # If using Cloud KMS
+        ```
+
+3. Setting up GCP credentials.
+
+    1. Create a service account within the GCP project.
+
+        ```shell
+        export SA_NAME=[Service Account Name]
+        gcloud iam service-accounts create ${SA_NAME}
+        ```
+
+    2. Add permissions to the created service account.
+
+        ```shell
+        # permission to create note
+        gcloud projects add-iam-policy-binding $PROJECT_ID \
+          --member serviceAccount:${SA_NAME}@${PROJECT_ID}.iam.gserviceaccount.com \
+          --role roles/containeranalysis.notes.editor
+
+        # permission to view vulnerability and attestation occurrences
+        gcloud projects add-iam-policy-binding $PROJECT_ID \
+          --member serviceAccount:${SA_NAME}@${PROJECT_ID}.iam.gserviceaccount.com \
+          --role roles/containeranalysis.notes.occurrences.viewer
+
+        # permission to upload attestation occurrences
+        gcloud projects add-iam-policy-binding $PROJECT_ID \
+          --member serviceAccount:${SA_NAME}@${PROJECT_ID}.iam.gserviceaccount.com \
+          --role roles/containeranalysis.occurrences.editor
+
+        # (if using Cloud KMS) permission to cloud KMS signing service
+        gcloud projects add-iam-policy-binding $PROJECT_ID \
+          --member serviceAccount:${SA_NAME}@${PROJECT_ID}.iam.gserviceaccount.com \
+          --role roles/cloudkms.signer
+        ```
+
+    3. Create json credentials for the service account.
+
+        ```shell
+        gcloud iam service-accounts keys create ./sa.json  --iam-account ${SA_NAME}@${PROJECT_ID}.iam.gserviceaccount.com
+        ```
+
+        The json credentials are saved in `sa.json`.
+
+    4. Setting GAC environment variable.
+
+        ```shell
+        export GOOGLE_APPLICATION_CREDENTIALS="$PWD/sa.json"
+        ```
+
+        Now the signer tool will automatically pick up the credentials via the environment variable.
+
+4. Creating a signing key. 
+
+    A private signing key is required to create attestations for an image. 
+Here, we use Cloud KMS as an example. The tool also supports PGP and PKIX keys.
```

Comment:
"Here, we use Cloud KMS as an example." -> "In this example you use Cloud KMS with PKIX keys. Kritis Signer also supports PGP keys."

Reply:
done.